src/core/svgRenderer.coffee

```diff
@@ -4,7 +4,7 @@ renderers = {}
 
```

Comment:
No prob :stuck_out_tongue:. I'm not a coffeescript guy, so if there's something wrong or something I can fix here please let me know. I only made these changes so I could actually get it to compile.